### ISSUE TITLE
Flush out Todo Entity

### ISSUE THREAD

**Issue by snaveevans** [honey-dos/web#7]
- [ ] Add properties
- [ ] Add tests

**Comment by HeedfulCrayon** [comment]
I would imagine that you would also was a reminder time (as in the time before the due date that the reminder would fire)

**Comment by snaveevans** [comment]
> I would imagine that you would also was a reminder time (as in the time before the due date that the reminder would fire)

Yes, let's implement that later though.  Because we don't have any sort of notification system set up, so a reminder time wouldn't trigger anything.

Did you want to try to tackle this one?
